src/drivers/intel/cavs/hda-dma.c

```diff
@@ -375,7 +409,7 @@ static int hda_dma_copy(struct dma *dma, int channel, int bytes, uint32_t flags)
 	else if (chan->state & HDA_STATE_PRELOAD)
 		return hda_dma_preload(dma, chan);
```

Comment:
How about renaming to hda_dma_host_preload() to indicate this is host only piece of code?